engine/src/assets/AssetImporterContext.cpp

```
@@ -0,0 +1,49 @@
+
+    const std::vector<GenericAssetRef>& AssetImporterContext::getDependencies() const
+    {
+        return m_dependencies;
+    }
+
+    void AssetImporterContext::setParameters(const nlohmann::json& params)
+    {
+        m_jsonParameters = params;
+    }
+
+    nlohmann::json AssetImporterContext::getParameters() const
```

Cppcheck (2.10-2)

[style] The function 'getParameters' is never used.

(unusedFunction)